client/src/www/app/outline_server_repository/server.ts

```diff
@@ -19,6 +19,7 @@ import * as errors from '../../model/errors';
 import * as events from '../../model/events';
 import {PlatformError} from '../../model/platform_error';
 import {Server, ServerType} from '../../model/server';
+import { ShadowsocksSessionConfig, VpnApi, TunnelStatus } from './vpn';
```

Comment:
I think this should be fixed by linting?

```suggestion
import {ShadowsocksSessionConfig, VpnApi, TunnelStatus} from './vpn';
```